contracts/DutchExchange.sol

```diff
@@ -1,16 +1,18 @@
 pragma solidity ^0.4.21;
 
 import "./TokenFRT.sol";
-import "./Oracle/PriceOracleInterface.sol";  
 import "@gnosis.pm/owl-token/contracts/TokenOWL.sol";
-import "@gnosis.pm/util-contracts/contracts/Proxy.sol";
+import "./base/TokenWhitelist.sol";
+import "./base/DxMath.sol";
```

Comment:
let's us a generic Math library.

If we use one, which uses internal function calls, then also there is no disadvantage gas cost vise.